test/SparkLendMainnetIntegration.t.sol

```diff
@@ -85,6 +92,9 @@ contract SparkLendMainnetIntegrationTest is Test {
     }
 
     function test_dai_market_irm() public {
+        // Set fork state to before this was introduced
+        vm.createSelectFork(getChain("mainnet").rpcUrl, 18784436);  // Dec 14, 2023
```

Comment:
I think it would be cleaner to update the tests, running them on a fork with a stale state. If there are only number changes to be made, it shouldn't be that much work to fix them, if there are bigger differences, then they should be updated anyway.